sec.club/src/App.js

```diff
@@ -4,6 +4,7 @@ import { createMuiTheme, ThemeProvider } from "@material-ui/core"
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
 import { red, cyan } from "@material-ui/core/colors";
 import Loader from "./components/Loader/Loader.js";
+import Footer from "./components/Footer/Footer.jsx"
```

Comment:
Put this with the header import

```suggestion
const Footer = lazy(() => import(/* webpackChunkName: "components" */"./components/Footer/Footer.jsx"));
```